src/rely/QCheck_rely.re

```diff
@@ -0,0 +1,42 @@
+module T = QCheck.Test;
+module Rely = Rely;
+
+let seed_ =
+  lazy {
+    let s =
+      try (int_of_string @@ Sys.getenv("QCHECK_SEED")) {
+      | _ =>
+        Random.self_init();
+        Random.int(1000000000);
+      };
+    /* Printf.printf("qcheck random seed: %d\n%!", s); */
+    s;
+  };
+
+let default_rand = () =>
+  /* random seed, for repeatability of tests */
+  Random.State.make([|Lazy.force(seed_)|]);
+
+let long_ =
+  lazy (
+    switch (Sys.getenv("QCHECK_LONG")) {
+    | "1"
+    | "true" => true
+    | _ => false
+    | exception Not_found => false
+    }
+  );
+
+let to_rely =
```

Comment:
Is there a reason we are not using camelCase for these functions? I feel like we should be keeping Rely conventions for naming since that is the intended audience.